conf/modules.config

```
@@ -883,4 +883,28 @@ process {
             enabled: true
         ]
     }
+
+    //
+    // SEXDETERRMINE
+    //
+    withName: SAMTOOLS_DEPTH {
+        tag = { "${meta1.reference}|${meta1.sample_id}_${meta1.library_id}" }
+        ext.prefix = { "${meta1.sample_id}_${meta1.library_id}_samtoolsdepth" }
+        publishDir = [
+            path: { "${params.outdir}/sex_determine/samtools_depth/" },
+            mode: params.publish_dir_mode,
+            enabled: true
+        ]
+    }
+
+    withName: SEXDETERRMINE {
+        tag = { "${meta.reference}|${meta.sample_id}_${meta.library_id}" }
+        ext.prefix = { "${meta.sample_id}_${meta.library_id}_sexdetermine" }
+        publishDir = [
+            path: { "${params.outdir}/sex_determine/sexdeterrmine/" },
+            mode: params.publish_dir_mode,
+            enabled: true
+        ]
```

```suggestion
        publishDir = [
            path: { "${params.outdir}/sex_determination/" },
            mode: params.publish_dir_mode,
            enabled: true
        ]
```
The output directory should be consistent with the one given in the output docs.

---

This might also need a `pattern`, to avoid posting the json as well as the TSV in the outputs.